moveit_core/robot_model/src/revolute_joint_model.cpp

```diff
@@ -163,7 +163,9 @@ bool RevoluteJointModel::satisfiesPositionBounds(const double* values, const Bou
   if (continuous_)
     return true;
   else
+  {
     return !(values[0] < bounds[0].min_position_ - margin || values[0] > bounds[0].max_position_ + margin);
+  }
```

Comment:
Thank you for this.  Can you also add the brackets around the if statement on the lines above?

Reply:
this was just leftover from debugging, but OK

Reply:
this was just leftover from debugging, but OK